spec/shared_helper.rb

```diff
@@ -34,6 +34,7 @@ def gemsets
   end
 
   def use_simplecov
+    return unless ENV['COVERAGE']
```

Comment:
Shouldn't the caller of `use_simplecov` decide whether simplecov is supposed to be used? If not, we should rename the method.